aten/src/ATen/core/DistributionsHelper.h

```diff
@@ -36,6 +36,7 @@
 
 
 namespace at {
+namespace {
```

Comment:
Hmm, should this one needs to be in the anonymous namespace or not?
Since it depends on `RNG`, which technically should be in its own anonymous namespace depending on the CPU architecure, there shouldn't be any name collisions between AVX and non-AVX code

Reply:
Pushed an update to `torch.float16` PR to see if Mac failures would still trigger if this is not in anonymous namespace

Reply:
This must be in anonymous namespace as well as https://app.circleci.com/pipelines/github/pytorch/pytorch/161351/workflows/82c92dea-3b9a-472f-83d7-221aeb0e8ef7/jobs/5283546/steps shows